### PR TITLE
keyword list

### PR DESCRIPTION
**Pull request by asem1789**
Relates #17

### CODE REVIEW

**Comment by aminalakhsham** [comment]
where the Relates ?!

File: client/src/components/Keywords/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const Keywords = ({ keyword, setKeyword }) => (
+  <div className="keywords">
+    <Icon className="keywords__close" type="close" />
+    <h2 className="keywords__title">Keywords</h2>
+    <div className="keywords__list">
+      {keywordsData.map((el, index) => {
+        return (
+          <Button
+            key={`keyword${index + 1}`}
```
Comment (mossa-Sammer) [inline client/src/components/Keywords/index.js]:
```suggestion
            key={element}
```

File: client/src/components/Keywords/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import Button from '../Button';
+
+import './index.css';
+
+const Keywords = ({ keyword, setKeyword }) => (
+  <div className="keywords">
+    <Icon className="keywords__close" type="close" />
+    <h2 className="keywords__title">Keywords</h2>
+    <div className="keywords__list">
+      {keywordsData.map((el, index) => {
```
Comment (mossa-Sammer) [inline client/src/components/Keywords/index.js]:
```suggestion
      {keywordsData.map((element, index) => {
```

File: client/src/components/Keywords/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        return (
+          <Button
+            key={`keyword${index + 1}`}
+            onClick={() => setKeyword(el.value)}
```
Comment (mossa-Sammer) [inline client/src/components/Keywords/index.js]:
```suggestion
            onClick={() => setKeyword(element.value)}
```

File: client/src/components/Keywords/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            className={
+              el.value === keyword
```
Comment (mossa-Sammer) [inline client/src/components/Keywords/index.js]:
```suggestion
              element.value === keyword
```

File: client/src/components/Keywords/index.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+                ? 'keywords__list--item active-btn'
+                : 'keywords__list--item'
+            }
+          >
+            {el.text}
```
Comment (mossa-Sammer) [inline client/src/components/Keywords/index.js]:
```suggestion
            {element.text}
```

**Pull request merged**